expression/builtin.go

```diff
@@ -731,3 +732,13 @@ func IsFunctionSupported(name string) bool {
 	_, ok := funcs[name]
 	return ok
 }
+
+// GetBuiltinList returns a list of builtin functions
+func GetBuiltinList() []string {
+	res := make([]string, 0, len(funcs))
+	for funcName := range funcs {
```

Comment:
not all the functions in `funcs` are implemented.

Reply:
Could you provide an example of such function?

Reply:
Only a few functions are not implemented now, for example, `builtinRowSig`.
When their evaluation methods are called, they cause a panic like `panic("xxx.evalXXX() should never be called")`.

Reply:
Thanks. Added an exclusion of some functions.